galaxykit/utils.py

```diff
@@ -0,0 +1,49 @@
+"""Utility functions"""
+import logging
+import json
+import time
+from urllib.parse import urljoin
+from ansible.galaxy.api import GalaxyError
```

Comment:
(opened https://github.com/ansible/ansible-hub-ui/pull/2355, let me know if that's the right fix please :))

Reply:
hmmm it looks like this was an added dependency, but I don't think it _needs_ to be. Am investigating now.

Reply:
More generally we've resisted adding a dependency on ansible-core, so I think if we can avoid it it's worth doing - the GalaxyError is only used down in the `wait_for_task` function (line 39) and I think since we aren't throwing a GalaxyError anywhere, it's fine for us to just catch any exception, rather than to specify the exception type.

Reply:
will have a PR up in just a few.

Reply:
right, thanks @hendersonreed !

Reply:
"just a few" was optimistic - was trying to force a task failure in order to verify that my changes worked as expected and then got wrapped up in the fact that my development environment is semi-broken. Will be working on it more today and hopefully have a PR up.